### PR TITLE
Styling tracker popup

### PR DESCRIPTION
**Pull request by chrisnojima**
This is the first pass at styling the tracker popup

This is a temp style so it can largely go away when we start really styling the app but its a good exploration of how we might want to do it for real.

Some things to note:
- Using font-awesome (all icons in the design are in there on purpose)
- Using inline styles (blasphemy!, see https://speakerdeck.com/vjeux/react-css-in-js ). I think this is the way to go as it aligns so much with the concept of a component as a really re-usable-drive-by-props thing

This does deviate from the designs a little, but it was all discussed with @malgorithms 

@keybase/react-hackers

### CODE REVIEW

**Comment by cjb** [comment]
:thumbsup:

**Comment by MarcoPolo** [comment]
What happened to all the type annotations in the render files?

**Comment by chrisnojima** [comment]
I'm going ot have to merge this with your branch. it's off of old master

File: react-native/react/tracker/proofs.render.desktop.js [hunk 1]
```diff
@@ -32,25 +33,47 @@ export default class ProofsRender extends Component {
   }
 
   renderPlatformProof (pp) {
-    const name = pp.platform.name === 'web' ? pp.platform.uri : pp.platform.name
-    console.log(name)
+    const name = pp.platform.name === 'web' ? pp.platform.uri : pp.platform.username
```
Comment (MarcoPolo) [inline react-native/react/tracker/proofs.render.desktop.js]: This was changed in my stuff. Remember we didn't want to do this kind of translation logic here

**Comment by MarcoPolo** [comment]
ah okay, lmk if you have any problems, but it should work if it type checks :)

**Comment by chrisnojima** [comment]
moving to this pr: https://github.com/keybase/client/pull/1404

**Pull request closed without merging**